Flask Visual Overhaul & YML Organizing

Prole0 opened the pull request:
<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->

## About the PR
<!-- What did you change? -->

Added Inhand/Open Sprites & Open Close Sounds to Flasks, organized the drinks_flask.yml, & slightly modified hos flask for its Inhand/Open Visuals.

## Why / Balance
<!-- Discuss how this would affect game balance or explain why it was changed. Link any relevant discussions or issues. -->

Mostly #33386 for Inhands, the Open visuals are for clarity, flasks now need to be open to be drunk from. The sounds are for distinction, bottleopen/closed sounds don't fit metal containers. 

Moved the metal pitcher to drinks_special & changed its Parent. Was confused on why a Pitcher was in the flask section, so I moved it. And I didn't want to accidentally make changes to the metal pitcher due to me changing the flask parent, which it was a child too. 

I also added a Full suffix to the MRE Flask, it spawns in with 50u of water, so I think it's fitting.

In my opinion, the hos flask's top part didn't mesh with the other flasks and generally looked odd. You can see the changes below in Media.

And I changed the name of the now old flask and added a description to it. Wanted the name to be similar to its file name & the description is to say that it has no lid.


## Technical details
<!-- Summary of code changes for easier review. -->

Inhands heavily based on Inhands supplied by #37771, thank you Shark.

Mostly Yaml & Json Changes, with a sprinkle of ogg. Attributions should be in place.

Made a Parent for icon_open Visuals named DrinkFlaskVisualsOpenable. It's what gives the flasks their Openable trait, I put it here rather than the FlaskBase due to the oldFlask. Wanted the oldFlask to function how flasks currently are, and it seems like its missing the lid part in its sprite. So, I put type: Openable in this Parent so I could be able to slot the Parent in if the Flask needed an open sprite.


## Media
<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->

Hos Flask Change (Old => New)
![Icon_Output](https://github.com/user-attachments/assets/d05db682-ebe9-4398-8939-0a77c06a2f4b)

Icon_Open Additions
![Icon_Open_Output](https://github.com/user-attachments/assets/7588204a-ea06-406b-8fe9-dcfbf145b05d)

Inhand Additions
![Inhands_Output](https://github.com/user-attachments/assets/c5dbc11d-94cc-44ee-90b1-1b38f99e947c)

Sound Example

https://github.com/user-attachments/assets/32c3bafd-2223-4fff-b16f-52197d206469

## Requirements
<!-- Confirm the following by placing an X in the brackets [X]: -->
- [X] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [X] I have added media to this PR or it does not require an ingame showcase.
<!-- You should understand that not following the above may get your PR closed at maintainer’s discretion -->

## Breaking changes
<!-- List any breaking changes, including namespaces, public class/method/field changes, prototype renames; and provide instructions for fixing them.
This will be posted in #codebase-changes. -->

**Changelog**
<!-- Add a Changelog entry to make players aware of new features or changes that could affect gameplay.
Make sure to read the guidelines and take this Changelog template out of the comment block in order for it to show up.
Changelog must have a :cl: symbol, so the bot recognizes the changes and adds them to the game's changelog. -->
<!--
:cl:
- add: Added fun!
- remove: Removed fun!
- tweak: Changed fun!
- fix: Fixed fun!
-->
:cl:
- add: Flask Visuals & Sounds!
- tweak: Flasks now need to be opened to be drunk from.

TiniestShark commented:
Solid stuff! I really dig the effort in making the icons openable too. New HoS flask looks great and good inhands overall! Only real naggle I can think of is making the symbol on the lithium flask properly square instead of a straight line so it matches up to the sprite better, and helps make it distinct from the others.
![1](https://github.com/user-attachments/assets/fb3ec57e-cbec-483f-914e-0e0e24a2923c)

Adjust that a bit and im happy to give this an approval!

Prole0 commented:
Adjusted, added the red near the center and changed the center to be light blue. All in place to be committed if nothing else is needed.

Urist Holding Revised Lith Flask
![New_Inhands_Output](https://github.com/user-attachments/assets/4484bd22-aa56-4f3a-9a45-c17b7dd8afe8)

inhand-left
![Inhand-Left](https://github.com/user-attachments/assets/5ce4c5e8-101c-47a1-a9e8-33ef110db41b)

inhand-right
![Inhand-Right](https://github.com/user-attachments/assets/7b9f21e2-556a-4192-997c-50b284d5a75a)

Prole0 commented:
> The copyright on the sound effects added looks good. I'm not so sure on the opening sound - I could take it or leave it. I'd prefer something a bit wetter... 
> 
> If you'd like, I actually have 2 flasks. They're twist-off tops instead of clickable ones, but I could record some open/close sound effects myself and send them your way. 

Oh for sure! I'm guessing through discord right? If so my @ is the same name.

NoahCHeaverin commented:
> > The copyright on the sound effects added looks good. I'm not so sure on the opening sound - I could take it or leave it. I'd prefer something a bit wetter... 
> > 
> > If you'd like, I actually have 2 flasks. They're twist-off tops instead of clickable ones, but I could record some open/close sound effects myself and send them your way. 
> 
> Oh for sure! I'm guessing through discord right? If so my @ is the same name. 

Mine is @darkicedcoffee! I've sent a friend request to @Prole on discord

Prole0 commented:
> Heya, just poking my head in here: Still working on getting that new sound, or would you rather stick with the one you're currently using?
> 
> The one you have sounds fine and there's no sound approval you need. I just need to get another maint to check the yaml and this'll be good to merge.

I'll continue with the current sounds. Thank you for the checkup.

beck-thompson reviewed (approved):
Looks good!
